eng/Versions.props

```diff
@@ -208,9 +208,10 @@
     -->
     <Analyzer_MicrosoftCodeAnalysisCSharpVersion>3.3.1</Analyzer_MicrosoftCodeAnalysisCSharpVersion>
     <Analyzer_MicrosoftCodeAnalysisCSharpWorkspacesVersion>3.3.1</Analyzer_MicrosoftCodeAnalysisCSharpWorkspacesVersion>
-    <MicrosoftCodeAnalysisCommonVersion>4.2.0-2.22128.1</MicrosoftCodeAnalysisCommonVersion>
-    <MicrosoftCodeAnalysisCSharpVersion>4.2.0-2.22128.1</MicrosoftCodeAnalysisCSharpVersion>
-    <MicrosoftCodeAnalysisCSharpWorkspacesVersion>4.2.0-2.22128.1</MicrosoftCodeAnalysisCSharpWorkspacesVersion>
+    <MicrosoftCodeAnalysisExternalAccessAspNetCoreVersion>4.3.0-3.22329.30</MicrosoftCodeAnalysisExternalAccessAspNetCoreVersion>
+    <MicrosoftCodeAnalysisCommonVersion>4.3.0-3.22329.30</MicrosoftCodeAnalysisCommonVersion>
+    <MicrosoftCodeAnalysisCSharpVersion>4.3.0-3.22329.30</MicrosoftCodeAnalysisCSharpVersion>
+    <MicrosoftCodeAnalysisCSharpWorkspacesVersion>4.3.0-3.22329.30</MicrosoftCodeAnalysisCSharpWorkspacesVersion>
```

Comment:
Will our dependency update flow clobber this?

Reply:
I don't know, but I hope they won't. I copied this version number from the Microsoft.CodeAnalysis assemblies that ship with .NET 7 SDK a few weeks ago.

The .NET 7 SDK is probably using new versions now, but that shouldn't be a problem.

Reply:
It's under `<PropertyGroup Label="Manual">` so it won't be overwritten automatically.

Reply:
Ah, I misunderstood. Yeah, these versions aren't updated automatically.

I'm more concerned about ASP.NET Core referencing a newer version than what's shipped with the SDK, but I've check the SDK versions so that won't happen.